src/models/piranha_arguments.rs

```diff
@@ -61,172 +68,237 @@ pub struct PiranhaArguments {
   #[clap(short = 'f', long)]
   #[serde(skip)]
   path_to_configurations: String,
+
   /// Path to output summary json file
   #[get = "pub"]
   #[builder(default = "default_path_to_output_summaries()")]
   #[clap(short = 'j', long)]
   #[serde(skip)]
   path_to_output_summary: Option<String>,
 
-  // a list of file extensions
-  // #[get = "pub"]
-  #[builder(default = "default_languages()")]
+  // the target language
+  #[builder(default = "default_language()")]
   #[clap(skip)]
-  #[serde(default = "default_languages")]
-  language: Vec<String>,
+  #[serde(default = "default_language")]
+  language: String,
```

Comment:
We are switching from a vector of languages to a single language. Does this mean that before we supported running a set of rules to fix point on multi-language projects? Do we plan to support that? For Android, this might be important, since there are java+kotlin projects both internally at Uber and more broadly externally, and it might not be the only case where it applies.

Reply:
Yes. So actually `language: Vec<String>`  was always unnecessary syntactic sugar.  I was using `language[0]` at the [getter](https://sourcegraph.com/github.com/uber/piranha/-/blob/src/models/piranha_arguments.rs?L141). 
I cleaned this up by migrating it from `Vec<String>` to `String`.

At this moment (or previously) we do not have support for 2 languages at once.
But internally we manage this mix language support by interleaving cleanup steps in each language.
This is still possible now (because each step essentially transforms code of one language). 
For instance, we run one step that updates `BUCK` files and follow that step up by migrating the API (`java` and `kotlin`). 

However, there are scenarios when interleaving at such coarse level will not be possible. We would want to invoke a Java rule after a Kotlin or vice versa. But we do not have support for this right now. 
I always wanted to do this, hence I had `language` as `vector`. But I guess its a bit lower on our priority list.